components/org.wso2.carbon.identity.organization.management.application/src/main/java/org/wso2/carbon/identity/organization/management/application/OrgApplicationManagerImpl.java

```diff
@@ -621,6 +627,57 @@ public void shareApplication(String ownerOrgId, String sharedOrgId, ServiceProvi
         }
     }
 
+    private boolean isOAuthClientExistsError(IdentityException e) {
+
+        return e.getErrorCode().equals(DUPLICATE_OAUTH_CLIENT.getErrorCode());
```

Comment:
```suggestion
        return DUPLICATE_OAUTH_CLIENT.getErrorCode().equals(e.getErrorCode());
```
Always use the pattern `constant.equals(variable)`, which reduce possible NPEs

Reply:
Note with thanks. 
Updated with [643826e](https://github.com/wso2-extensions/identity-organization-management/pull/330/commits/643826ed9f487f690a7ac0621c38e22b1a8570a5)